index.bs

```diff
@@ -638,3 +646,134 @@ registerAnimator('spring', class {
     }
 });
 </pre>
+
+
+Example 2: Twitter header. {#example-2}
+--------------------------
+
+
+<pre class='lang-markup'>
+&lt;style&gt;
+#scroller {
+  --animator-root: twitter-header;
+}
+.avatar {
+  --animator: twitter-header avatar;
+}
+.bar {
+  --animator: twitter-header bar;
+}
+&lt;/style&gt;
+
+&lt;div id="scroller"&gt;
+  &lt;section class="bar"&gt;&lt;/section&gt;
+  &lt;header&gt;
+    &lt;picture class="avatar"&gt;
+      &lt;img src="avatar.jpg"&gt;
+    &lt;/picture&gt;
+    &lt;section&gt;
+      &lt;button&gt;Friends&lt;/button&gt;
+      &lt;button&gt;Edit Profile&lt;/button&gt;
+    &lt;/section&gt;
+  &lt;/header&gt;
+&lt;/div&gt;
+
+&lt;script&gt;
+document.animationWorklet.import('tweeter-header.js');
+&lt;/script&gt;
+
+</pre>
+
+<pre class='lang-javascript'>
+// Inside AnimationWorkletGlobalScope.
+registerAnimator('twitter-header', class {
+
+  static elements =  [
+    {name: 'avatar', inputProperties: [], outputProperties: ['transform']},
+    {name: 'bar', inputProperties: [], outputProperties: ['opacity']}
+  ];
+
+  static timelines = [ 'scroll' ];
+
+  static INIT_HEIGHT = 140;
+
+  animate(elementMap, timelines) {
+    var scrollPos = timelines[0].currentTime;
```

Comment:
I think we can't pass in the raw scroll offsets through the scrolling timeline because as we discussed it maps from [startScrollOffset, endScrollOffset] to [0, timeRange]. Check out the conversion in https://googlechrome.github.io/houdini-samples/animation-worklet/twitter-header/twitter-header-animator.js (note this example should have timeRange set to 1, otherwise as an infinite duration animation the output is just 0).